microservice/src/main/java/com/skytouch/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.skytouch.service;
+
+
+import com.skytouch.model.Product;
+import com.skytouch.model.ProductEntity;
+import com.skytouch.repository.ProductRepository;
+import com.skytouch.utility.Mapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Service
+class ProductServiceImpl implements ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductServiceImpl(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    @Override
+    public List<Product> getProducts() {
+        List<Product> result =
```

Comment:
You could return the result of the operation without having to assign its value to a variable